pkgs/games/dwarf-fortress/default.nix

```diff
@@ -35,6 +47,13 @@ let
   dfHackWorksWithCurrentVersion = true;
   dfHackEnabled = dfHackWorksWithCurrentVersion && enableDFHack;
 
+  themesThatWorkWithCurrentVersion = [
```

Comment:
I'm not sure if I quote understand what that means, but I'll try and explain the reasoning behind this piece of code. When Dwarf Fortress updates, the themes may not be updated immediately, since they are created by contributors rather than the game developer himself.

Therefore, if we update the dwarf fortress package version, we can simply remove any theme from this list that hasn't updated, and the game will at least run fine. This prevents the game updates from being held back by extras. Once an updated theme is released, we add the theme back to this list.